src/modules/attitude_estimator_q/attitude_estimator_q_main.cpp

```diff
@@ -305,6 +286,9 @@ void AttitudeEstimatorQ::task_main()
 			}
 
 			_data_good = true;
+
+		} else {
+			continue;
```

Comment:
Since this might become an example on how to do it, can we invert the logic like this:

```
if (!_sensors_sub.updateBlocking(sensors)) {
    continue;
}

// do work...
```

Reply:
Done.